core/lib/zksync_core/src/api_server/web3/tests/ws.rs

```diff
@@ -72,7 +74,10 @@ trait WsTest {
     ) -> anyhow::Result<()>;
 }
 
-async fn test_ws_server(test: impl WsTest) {
+async fn test_ws_server(
+    test: impl WsTest,
+    websocket_requests_per_minute_limit: Option<NonZeroU32>,
```

Comment:
Bikeshedding: Instead of introducing this parameter, you could extend `WsTest` with the corresponding getter:

```rust
trait WsTest {
    fn websocket_requests_per_minute_limit(&self) -> Option<NonZeroU32> {
        None
    }
    // ...
}
```

In this way, less existing code would be changed.

Reply:
done